### ISSUE TITLE
management commands for badges are not working

### ISSUE THREAD

**Issue by alexsavio** [EuroPython/epcon#1047]
I need the manage commands `ticket_profiles` and `talk_abstracts` to get enough metadata to generate the badges.

When I try to run the first I get this:
```Traceback (most recent call last):
  File "manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 206, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 40, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/epstage/p3/management/commands/ticket_profiles.py", line 18, in <module>
    class Command(BaseCommand):
  File "/usr/src/epstage/p3/management/commands/ticket_profiles.py", line 19, in Command
    option_list = BaseCommand.option_list + (
AttributeError: type object 'BaseCommand' has no attribute 'option_list'
```

when I try to run `talk_abstracts`, I get:

```
Traceback (most recent call last):
  File "manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 206, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 40, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/ep2019/p3/management/commands/talk_abstracts.py", line 27, in <module>
    class Command(BaseCommand):
  File "/usr/src/ep2019/p3/management/commands/talk_abstracts.py", line 28, in Command
    option_list = BaseCommand.option_list + (
AttributeError: type object 'BaseCommand' has no attribute 'option_list'
Traceback (most recent call last):
  File "/home/alexandre/.pyenv/versions/epcon/bin/inv", line 10, in <module>
    sys.exit(program.run())
  File "/home/alexandre/.pyenv/versions/3.7.3/envs/epcon/lib/python3.7/site-packages/invoke/program.py", line 363, in run
    self.execute()
  File "/home/alexandre/.pyenv/versions/3.7.3/envs/epcon/lib/python3.7/site-packages/invoke/program.py", line 532, in execute
    executor.execute(*self.tasks)
  File "/home/alexandre/.pyenv/versions/3.7.3/envs/epcon/lib/python3.7/site-packages/invoke/executor.py", line 129, in execute
    result = call.task(*args, **call.kwargs)
  File "/home/alexandre/.pyenv/versions/3.7.3/envs/epcon/lib/python3.7/site-packages/invoke/tasks.py", line 128, in __call__
    result = self.body(*args, **kwargs)
  File "/home/alexandre/projects/europython/ep-tools/tasks.py", line 137, in fetch_talks_json
    with_votes=with_votes
  File "/home/alexandre/projects/europython/ep-tools/eptools/talks/fetch.py", line 39, in fetch_talks_json
    talks = json.load(f)
  File "/home/alexandre/.pyenv/versions/3.7.3/lib/python3.7/json/__init__.py", line 296, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/home/alexandre/.pyenv/versions/3.7.3/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/home/alexandre/.pyenv/versions/3.7.3/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/alexandre/.pyenv/versions/3.7.3/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by alexsavio** [comment]
Maybe there are substitutes for these scripts elsewhere?

**Comment by umgelurgel** [comment]
Fixed in #1048
